extensions/wikia/Flags/controllers/FlagsApiController.class.php

```diff
@@ -325,6 +339,9 @@ private function getFlagTypesForWikiaRawData( $wikiId ) {
 			$flagTypesForWikia = $flagTypeModel->getFlagTypesForWikia( $wikiId );
 
 			$flagsCache->setFlagTypesForWikia( $flagTypesForWikia );
+			$this->logCache( 'getFlagsForPageForEdit', 'MISS' );
+		} else {
+			$this->logCache( 'getFlagsForPageForEdit', 'HIT' );
```

Comment:
If 1% sampling is fine for you, we do collect memcache stats (number of hits, misses, gets, sets) in Influx DB.